jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/BuildDockerTask.java

```diff
@@ -22,60 +22,23 @@
 import com.google.cloud.tools.jib.frontend.CacheDirectoryCreationException;
 import com.google.cloud.tools.jib.frontend.HelpfulSuggestions;
 import com.google.cloud.tools.jib.http.Authorization;
-import com.google.cloud.tools.jib.http.Authorizations;
-import com.google.cloud.tools.jib.image.ImageReference;
 import com.google.cloud.tools.jib.image.InvalidImageReferenceException;
 import com.google.cloud.tools.jib.registry.credentials.RegistryCredentials;
 import com.google.common.base.Preconditions;
 import java.nio.file.Path;
 import javax.annotation.Nullable;
 import org.gradle.api.DefaultTask;
 import org.gradle.api.GradleException;
-import org.gradle.api.tasks.Nested;
 import org.gradle.api.tasks.TaskAction;
 
 /** Builds a container image and exports to the default Docker daemon. */
 public class BuildDockerTask extends DefaultTask {
 
-  /**
-   * Directory name for the cache. The directory will be relative to the build output directory.
-   *
-   * <p>TODO: Move to ProjectProperties.
-   */
-  private static final String CACHE_DIRECTORY_NAME = "jib-cache";
-
   private static final HelpfulSuggestions HELPFUL_SUGGESTIONS =
       HelpfulSuggestionsProvider.get("Build to Docker daemon failed");
 
-  /**
-   * Converts an {@link ImageConfiguration} to an {@link Authorization}.
-   *
-   * <p>TODO: Move to ImageConfiguration.
-   */
-  @Nullable
-  private static Authorization getImageAuthorization(ImageConfiguration imageConfiguration) {
-    if (imageConfiguration.getAuth().getUsername() == null
-        || imageConfiguration.getAuth().getPassword() == null) {
-      return null;
-    }
-
-    return Authorizations.withBasicCredentials(
-        imageConfiguration.getAuth().getUsername(), imageConfiguration.getAuth().getPassword());
-  }
-
   @Nullable private JibExtension jibExtension;
 
-  /**
```

Comment:
Why was this method removed?

Reply:
Ah, I think the IDE gave an "unused" warning and I wasn't paying attention.

Reply:
Ah yea IntelliJ is not intellijent enough to recognize stuff like this